<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>859</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Podstawowy kalkulator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font-family: Arial;
	font-size: 24px;
}
#centralwidget{
	background-color: #0ca678;
}
QPushButton{
	height: 60px;
	border: 2px solid white;
	border-radius: 12px;
	color: white;
	background-color: transparent;
}
QPushButton:hover{
background-color: white; 
color: #0ca678;
}
QLabel{
	padding-right: 10px;
	border: 2px solid white;
	border-radius: 12px;
	color: white;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>300</y>
      <width>421</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>7</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="2" column="3">
      <widget class="QPushButton" name="btnSub">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="btnMul">
       <property name="text">
        <string>×</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="btnDot">
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="btnDiv">
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="btnAdd">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="btnEqual">
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="btn7">
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="btn5">
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="btn1">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="btn4">
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="btn6">
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btn8">
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btn9">
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="btn0">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="btn2">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="btn3">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="btnClear">
       <property name="text">
        <string>C</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btnAllClear">
       <property name="text">
        <string>AC</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btnParentsClose">
       <property name="text">
        <string>)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="btnParentsOpen">
       <property name="text">
        <string>(</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="labelResult">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>401</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align: right</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBack">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
border-radius: 0;</string>
    </property>
    <property name="text">
     <string>Wróć</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelError">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>350</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
color: #ff6b6b;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
